src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -676,34 +676,34 @@ void BaseCouplingScheme::initializeTXTWriters()
       if (convMeasure.level > 0)
         hasCoarseModelOptimization = true;
 
-    _iterationsWriter->addData("Timesteps", io::TXTTableWriter::INT);
-    _iterationsWriter->addData("Total_Iterations", io::TXTTableWriter::INT);
+    _iterationsWriter->addData("Timestep", io::TXTTableWriter::INT);
+    _iterationsWriter->addData("TotalIterations", io::TXTTableWriter::INT);
     _iterationsWriter->addData("Iterations", io::TXTTableWriter::INT);
     if (hasCoarseModelOptimization) {
-      _iterationsWriter->addData("Total_Iterations_Surrogate_Model", io::TXTTableWriter::INT);
-      _iterationsWriter->addData("Iterations_Surrogate_Model", io::TXTTableWriter::INT);
+      _iterationsWriter->addData("TotalIterations_SurrogateModel", io::TXTTableWriter::INT);
+      _iterationsWriter->addData("Iterations_SurrogateModel", io::TXTTableWriter::INT);
     }
     _iterationsWriter->addData("Convergence", io::TXTTableWriter::INT);
 
     if (not doesFirstStep()) {
       _convergenceWriter->addData("Timestep", io::TXTTableWriter::INT);
-      _convergenceWriter->addData("Iteration", io::TXTTableWriter::INT);
+      _convergenceWriter->addData("Iterations", io::TXTTableWriter::INT);
     }
 
-    int i = -1;
     if (not doesFirstStep()) {
-      for (ConvergenceMeasure &convMeasure : _convergenceMeasures) {
-        i++;
+      for (size_t i = 0; i < _convergenceMeasures.size(); i++) {
```

Comment:
Why converting from the ranged-for (previous) to the "classic" for-loop? We don't need the counter `i` anywhere and a ranged-for is safer.